apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -56,6 +56,8 @@
 
   SAMPLE_A_PDE(resourceUrl("rif-static-samples/sample-a-pde.txt"), RifFileType.PDE, 1),
 
+  SAMPLE_A_PDE_UPIN(resourceUrl("rif-static-samples/sample-a-pde-upin.txt"), RifFileType.PDE, 1),
```

Comment:
I think you can lose this? Why not just load the normal `SAMPLE_A` data in your test and then modify the field to have the value you want to test? (I think those record classes are all mutable, right?)

Reply:
your right :(